src/projects/Search/search.js

```diff
@@ -66,46 +66,22 @@ const getFormattedAttributes = function (attribute) {
 }
 
 const getPlaceholder = function (attribute, flag = false) {
-  if (attribute == 'allergy') {
-    return 'Allergies'
-  }
-  if (attribute == 'diets') {
-    return 'Specific Diets'
-  }
-  if (attribute == 'cuisine') {
-    return 'Specific Cuisine '
-  }
-  if (attribute == 'course') {
-    return 'Course'
-  }
-  if (attribute == 'holidays') {
-    return 'Holiday'
-  }
-  // case for ingredients, not for attributes...
-  if (attribute == 'ingredient') {
-    return (flag) ? 'Ingredients you have' : "Ingredients you don't have"
+  switch (attribute) {
```

Comment:
I think here you got mistaken. Compare old version of code with your changes and you'll see the issue. looks like i confuse you and instead of returning strings, you return parsed files

Reply:
i still don't think it's fixed.
take a look on old code
```
  if (attribute == 'allergy') {
    return 'Allergies'
  }
  if (attribute == 'diets') {
    return 'Specific Diets'
  }
  if (attribute == 'cuisine') {
    return 'Specific Cuisine '
  }
  if (attribute == 'course') {
    return 'Course'
  }
  if (attribute == 'holidays') {
    return 'Holiday'
  }
```
actually it's a good example of why having this habit - instead of deleting everything - it's better to comment it first and if everything is working great and approved - delete it later

Reply:
oh okay, I have typos in the strings so if typos are corrected this will solve the problem?

Reply:
maybe - show me updated version and i'll be able to tell you